docs/docs/rest-api/public/api/api.raml

```diff
@@ -0,0 +1,29 @@
+#%RAML 0.8
+
+title: Marathon REST API
+version: 2.0
```

Comment:
This value is not rendered in the output html. The entries are rather self-explanatory since they include the `/v2/` path element, but I think this should be used to render `Marathon REST API 2.0`